.github/workflows/release.yml

```diff
@@ -64,7 +68,7 @@ jobs:
           SLACK_MSG_AUTHOR: ${{ github.event.pull_request.user.login }}
           SLACK_USERNAME: GithubGoose
           SLACK_ICON_EMOJI: ":goose:"
-          SLACK_MESSAGE: "A new version of ${github.event.repository.name} was published! 🎉 Release notes → https://github.com/Khan/${github.event.repository.name}/releases/"
+          SLACK_MESSAGE: "A new version of ${{ github.event.repository.name }} was published! 🎉 \n${{ UPDATED_PACKAGES }}\nRelease notes → https://github.com/Khan/${{ github.event.repository.name }}/releases/"
```

Comment:
This fixes the string insertion issues we had and adds our nicely formatted list of packages too.

Reply:
ah that was the issue! thanks for fixing it.